app/models/find_api/course.rb

```diff
@@ -15,6 +15,12 @@ def name_and_code
       "#{name} (#{code})"
     end
 
+    def subject_codes
+      subjects.to_a.map do |subject|
+        subject.try(:attributes).try(:[], 'subject_code')
```

Comment:
Would this work?

```suggestion
        subject.dig('attributes', 'subject_code')
```

Reply:
sadly not, subject is a `FindAPI::Provider::Subject` that doesn't respond to `dig`